snapcraft/cli/_errors.py

```diff
@@ -55,3 +72,20 @@ def exception_handler(exception_type, exception, exception_traceback, *,
             echo.error(str(exception))
 
     sys.exit(exit_code)
+
+
+def _is_environment_ok():
```

Comment:
I don't like the name of this function.
What about: `_send_error_data()`?

Reply:
But it doesn't encompass everything